main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -666,7 +668,8 @@ object Resolver {
             case NamedAst.DefOrSig.Sig(sig) => visitApplySig(app, sig, exps, region, innerLoc, outerLoc)
           }
 
-        case app@NamedAst.Expression.Apply(_, _, _) => visitApply(app, region)
+        case app@NamedAst.Expression.Apply(_, _, _) =>
```

Comment:
Remove app@ binding

Reply:
Wait, sorry, I am confused. What is happening here.

Reply:
I guess the code was already like that.

Reply:
https://github.com/flix/flix/pull/4327#discussion_r928801498

Reply:
Yeah exactly :+1: